pkg/synchromanager/clustersynchro_manager.go

```diff
@@ -398,6 +446,17 @@ func (manager *Manager) reconcileCluster(cluster *clusterv1alpha2.PediaCluster)
 	return controller.NoRequeueResult
 }
 
+func (manager *Manager) stopClusterSynchro(name string) {
+	manager.synchrolock.Lock()
+	synchro := manager.synchros[name]
+	delete(manager.synchros, name)
+	manager.synchrolock.Unlock()
+
+	if synchro != nil {
+		synchro.Shutdown(false)
```

Comment:
Here you need to use true, to trigger a status update for the pediacluster.

This is not the same as `removeCluster`.